src/dmi.rs

```diff
@@ -0,0 +1,24 @@
+use std::fs::File;
+use png::{Decoder, Encoder, HasParameters};
+
+use error::Result;
+
+byond_fn! { strip_dmi_metadata(path) {
```

Comment:
I'm trying to prefix functions with their relevant module, so this should be `dmi_strip_metadata`, or similar.